sdk/appservice/arm-appservice-rest/samples-dev/appServiceEnvironmentsGetInboundNetworkDependenciesEndpointsSample.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
```

Comment:
Are these autogenerated?

Reply:
removed this comment as this is actually not auto generated